raw_data_processor/src/main/scala/scalalab3/lyricsengine/reader/MSD.scala

```diff
@@ -0,0 +1,9 @@
+package scalalab3.lyricsengine.reader
+
+
+/**
+  * That class represent data from Million Song Dataset (MSD).
+  */
+case class MSD(words: Seq[String], songs: Seq[Song])
```

Comment:
Do we have some limits on type/variable names? :) Why MSD, and not MillionSongDataset? 

And a bit more general question? What will happen if we realize in a month, that this Millon Song Dataset does not conform all our needs? And, lets imagine we will find (or generate) Billon Songs Dataset (another metadata, or at least size, which could not be loaded to RAM directly)?  So, could you please comment a bit about the idea of this class?